mathjax3-ts/output/svg/Wrappers/mtable.ts

```diff
@@ -191,118 +236,95 @@ CommonMtableMixin<SVGmtd<N, T, D>, SVGmtr<N, T, D>, SVGConstructor<N, T, D>>(SVG
      * @return {N}             The SVG element for the frame
      */
     protected makeFrame(w: number, h: number, d: number, style: string) {
-        const properties: OptionList = {
-            'data-frame': true,
-            width: this.fixed(w), height: this.fixed(h+d), y: this.fixed(-d)
-        }
-        if (style !== 'solid') {
-            properties.style = {'outline-style': style};
-        }
-        return this.svg('rect', properties);
+        const t = this.fLine;
+        return this.svg('rect', this.setLineThickness(t, style, {
+            'data-frame': true, 'class': 'mjx-' + style,
```

Comment:
Is there no method for adding the prefix? Or a constant that defines the prefix? Just in the event we ever want to change it in the future.